run.go

```diff
@@ -37,6 +37,9 @@ type PlaywrightDriver struct {
 
 func NewDriver(options *RunOptions) (*PlaywrightDriver, error) {
 	baseDriverDirectory := options.DriverDirectory
+	if baseDriverDirectory == "" {
+		baseDriverDirectory = os.Getenv("PLAYWRIGHT_DRIVER_PATH")
```

Comment:
nit: I'd probably put it into into the first line of getDefaultCacheDirectory, then the caller side is simpler. Either the user given value or our internal logic.

Reply:
I put it in `transformRunOptions` and added some comments. `getDefaultCacheDirectory` still maintains its single purpose.

Users can now understand the usage through the type comments of `RunOptions`.